openstudio-extension.gemspec

```diff
@@ -26,10 +26,10 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.5.0'
+  spec.required_ruby_version = '~> 2.7.0'
 
   spec.add_dependency 'bcl', '~> 0.6.1'
```

Comment:
Is this still relevant? Should it be a development_dependency instead?

https://github.com/NREL/bcl-gem/issues/8